README.md

```diff
@@ -29,15 +29,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+Map creates a new array.
 2. What is the difference between a function and a method?
-
+a method is a function inside of a function.
 3. What is closure?
-
+closure is the thing that an object needs to create a local scope.
```

Comment:
A closure is a nested function. Look deeper into 'closures' with 'You Don't know JS' book series available for free on Git Hub. I think this will help bring some clarity for you.